### DISCUSSION TITLE
Unable to enable TLSv1

### DISCUSSION THREAD

**Discussion by xXPerditorXx** [openssl/openssl#23565] [Q&A]
I'm unable to enable TLS version 1 on my arch Linux.
I edited the OpenSSL (3.2.1 30 Jan 2024)
conf at `/etc/ssl/openssl.cnf`
```
[openssl_init]
ssl_conf = ssl_sect

[ssl_sect]
system_default = system_default_sect

[system_default_sect]
CipherString = DEFAULT:@SECLEVEL=0
```

It's now my problem since about a year and back then it wasn't fixed either.

**Comment by mattcaswell** [comment]
What happens when you you try to use TLSv1?

Are you using the OpenSSL command line application? If so what command did you use and what errors do you see?

Or are you using your own application that is linked against libssl/libcrypto? In which case please tell us more about your application and what errors you see.

**Reply by xXPerditorXx** [reply]
I tried to connect to my school network because the city is doesn't know how to update to a newer tls.
It's called "radius" and is WPA Enterprise

**Comment by xXPerditorXx** [comment]
I noticed my [other discussion](https://github.com/openssl/openssl/discussions/22642) is still open too, which involves the same bug
